README.md

```diff
@@ -1 +1,38 @@
-# go-clang-phoenix-bootstrap
+# go-clang-phoenix-bootstrap [![GoDoc](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap?status.png)](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap) [![Build Status](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap.svg?branch=master)](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap) [![Coverage Status](https://coveralls.io/repos/zimmski/go-clang-phoenix-bootstrap-gen/badge.png?branch=master)](https://coveralls.io/r/zimmski/go-clang-phoenix-bootstrap?branch=master)
+
+Native Go bindings for the C API of Clang.
+
+#Usage
+
+There is no direct way to use this repo other than for bootstrapping new go-clang-bindings. 
+Look over to [gen](https://github.com/zimmski/go-clang-phoenix-gen) to discover how to generate a new go-clang-binding.
+
+## Where are the bindings?
```

Comment:
`I need bindings for a different Clang version`

Also, just link to https://github.com/zimmski/go-clang-phoenix-gen#where-are-the-bindings Listing all versions here would mean that we have to recreate each version repository for just one new version.